blobfile/_ops_test.py

```diff
@@ -1435,6 +1435,46 @@ def test_azure_etags(ctx):
         bf.remove(path)
 
 
+@pytest.mark.parametrize("ctx", [_get_temp_as_path])
+def test_azure_etags_last_version(ctx):
+    part1 = b"hello"
+    part2 = b" world"
+
+    with ctx() as path:
+        with bf.BlobFile(path, "wb", streaming=True) as f:
+            f.write(part1)
+            partial_written_version = bf.last_version_seen(f)

```

Comment:
```suggestion
            assert bf.last_version_seen(f) is None

```

Reply:
Done.

Reply:
The context already always deletes in advance